### ISSUE TITLE
AttributeError: module 'tensorflow' has no attribute 'Session'

### ISSUE THREAD

**Issue by tjf801** [HyperGAN/HyperGAN#245]
When I try to run hypergan, I get an error.

```
C:\Users\[censored]>hypergan test
2019-12-09 15:14:17.990916: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
Loading configuration C:\Program Files\Python37\lib\site-packages\hypergan\configurations\default.json
Hooray!
Hypergan is installed correctly.  Testing tensorflow for GPU support.
Traceback (most recent call last):
  File "C:\Program Files\Python37\Scripts\\hypergan", line 135, in <module>
    gancli.run()
  File "C:\Program Files\Python37\lib\site-packages\hypergan\cli.py", line 215, in run
    with tf.Session() as sess:
AttributeError: module 'tensorflow' has no attribute 'Session'
```

If I could have any tips or workarounds on what to do, that would be great. Thanks

**Comment by crazygmr101** [comment]
Can confirm this exact problem. Latest packages, Linux Mint 18.04, Python 3.7.

**Comment by theokktok** [comment]
I can reproduce this on Ubuntu 18.04 using Tensorflow 2.0 and Python 3.7.4.

Fixed downgrading Tensorflow to version 1.15.0 using "pip install tensorflow==1.15.0"

**Comment by catyoop** [comment]
had the same error, fixed it by doing the same as above, but went with tensorflow 1.13.1
pip install tensorflow==1.13.1
tensorflow 2.0 doesn't seem to work

**Comment by martyn** [comment]
This should be fixed in hypergan 1.0 when it's released (uses pytorch now).
